compiler/control/OMROptions.cpp

```diff
@@ -1016,6 +1016,10 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
      "M\tdo not activate another compilation thread when high priority request is blocked", RESET_OPTION_BIT(TR_ActivateCompThreadWhenHighPriReqIsBlocked), "F", NOT_IN_SUBSET },
     { "dontAddHWPDataToIProfiler", "O\tDont add HW Data to IProfiler", SET_OPTION_BIT(TR_DontAddHWPDataToIProfiler),
      "F", NOT_IN_SUBSET },
+    { "dontCompile=",
+     "O{regex}\t regex which specifies a subset of methods not to compile, if the option is passed via -Xaot, then it is not AOT compiled, but may be jit compiled. The converse is true for -Xjit",
```

Comment:
I suggest using a prefix of 'D' (for debugging options) rather than 'O' (for optimizer options).
Also, please add a period after "not to compile" instead of a comma.